docs/en/cloud/security/secure-kinesis.md

```diff
@@ -0,0 +1,84 @@
+---
+slug: /en/cloud/manage/security/secure-kinesis
+sidebar_label: Kinesis Role-based Access
+title: Kinesis Role-based Access
+---
+
+This article demonstrates how ClickHouse Cloud customers can leverage role-based access to authenticate with Amazon Kinesis and access their data streams securely.
```

Comment:
IMO docs should be under `clickpipes`. ClickHouse Cloud can't work with Kinesis directly (unlike with s3)